K8S.md

```diff
@@ -36,7 +36,13 @@ SSH into a node
 ```                                                                        
 cluster-up/ssh.sh node01                                                   
 ```                                                                        
-                                                                           
+
+Use slim provider (without cached images)  
+In order to save time downloading the image, set the following before cluster-up
```

Comment:
Maybe worth mentioning, in which use-cases this is beneficial - IUUC typically when extra components like CNAO,CNI,Istio are note required.

Otherwise pulling these separately after cluster-up wouldn't really save much, compared to pulling them as part of kubevirtci image.

Reply:
updated to use short explanation
bit more info is on KUBEVIRTCI_LOCAL_TESTING.md